modules/images-configuration-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuring-image-registry-settings-hcp_{context}"]
+= Configuring image registry settings for ROSA with HCP
```

Comment:
Customers will have the registry configuration options that they expect to have in a typical kubernetes/OpenShift based cluster, and can do so with some security measures being met by these capabilities.

Reply:
discuss tomorrow

Reply:
@chenz4027 are we ok to resolve this comment now?